### PR TITLE
Add setup wizard for docker containers

### PR DESCRIPTION
**Pull request by csstaub**
r: @mcpherrinm @alokmenghrajani 

Add setup wizard for docker containers.

Run the wizard like this:

    docker run -it \
        -v keywhiz-data:/data \
        -v keywhiz-secrets:/secrets \
        square/keywhiz wizard

The wizard will create symmetric encryption keys and ask for TLS certificates/keys.

After it completes, run keywhiz server with the generated config like this:
    
    docker run \
        -p 4444:4444
        -v keywhiz-data:/data \
        -v keywhiz-secrets:/secrets \
        -e 'KEYWHIZ_CONFIG=/data/keywhiz-docker.yaml'
        square/keywhiz server

### CODE REVIEW

File: docker/wizard.sh [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+export CONTENT_KEYSTORE_PATH="/secrets/content-encryption-key.jceks"
+rm -f $CONTENT_KEYSTORE_PATH
+keytool \
+    -genseckey -alias basekey -keyalg AES -keysize 128 -storepass "$CONTENT_KEYSTORE_PASSWORD" \
+    -keypass "$CONTENT_KEYSTORE_PASSWORD" -storetype jceks -keystore $CONTENT_KEYSTORE_PATH
+echo "done"
+
+echo
+echo "Keywhiz requires a SSL/TLS server certificate and private key to run."
+echo "You can import your own cert, or have the wizard generate one."
```
Comment (mcpherrinm) [inline docker/wizard.sh]: Can the wizard generate one? I don't see this

Comment (csstaub) [inline docker/wizard.sh]: Oh, whoops, no; I dropped that functionality. It seemed weird to generate it in the container and then dump the private key (which would be necessary to to make client certs). I'd rather just have people run that on their own.

Comment (mcpherrinm) [inline docker/wizard.sh]: :+1:

**Comment by mcpherrinm** [comment]
:shipit:

**Pull request merged**